### PR TITLE
create blog models

### PR DESCRIPTION
**Pull request by Becky449**
- Create a new app. Give it a meaningful name.
- Initialize your project with Git.
- Make sure that your project has Postgres database set up.
- Build your project schema.
- Create and run the necessary migration files.
- Table and column names should match the ERD diagram.
- note: photo for users table should be a link to an image
- Foreign keys should be included.
- Pay attention that in posts table the column holding the foreign key to the users table should be named author_id
- All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

**Review by zainsadaqat — COMMENTED** [review]
# ❌ ❌ ❌Invalid Code Review Request ❌ ❌ ❌
# Hello @Becky449 👋  

## Something went wrong 

### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for this reason:

- ### You have submitted a project, where you haven't set up any linters for the project. Please set up relevant linters for this project as it's part of the project requirement. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by FrederickMih — COMMENTED** [review]
## Hi :raising_hand:  @Becky449, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: 

Let's kindly remove the **Live Demo** header in the README.md file since the project hasn't been deployed.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: db/migrate/20230504181957_create_likes.rb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 class CreateLikes < ActiveRecord::Migration[7.0]
   def change
     create_table :likes do |t|
```
Comment (Davidosky007) [inline db/migrate/20230504181957_create_likes.rb]: * As shown in the ERD diagram you should have a foreign key that will create author_id and a corresponding index
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Becky449 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Well written PR 👍 
* Good looking readme 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230504181957_create_likes.rb (hunk 1 above)
Comment (rotimiazeez) [reply db/migrate/20230504181957_create_likes.rb]: As requested by the previous CR, you should include a foreign_key to your migrations that will link tables to each other. For example in your CreatePost migration, you should have something similar to the following to link the post to a user:
```
`t.references :author, foreign_key: { to_table: 'users' }`
```
- Now using the ERD diagram include a corresponding foreign_key to other migrations.

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
Hi @Becky449 😉, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Review by rustam-tolipov — APPROVED** [review]
Hi @Becky449, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Well-implemented changes 💯 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**